src/pages/billing/Project.js

```diff
@@ -14,187 +14,176 @@ import * as StateHistory from 'src/libs/state-history'
-              })
-            }, _.filter(({ email }) => Utils.textMatch(filter, email), projectUsers))
-          )
+          h(IdContainer, [id => h(Fragment, [
+            h(FormLabel, { htmlFor: id, required: true }, ['Select billing account']),
+            h(Select, {
+              id,
+              value: selectedBilling || billingAccountName,
+              isClearable: false,
+              options: _.map(({ displayName, accountName }) => ({ label: displayName, value: accountName }), billingAccounts),
+              onChange: ({ value: newAccountName }) => setSelectedBilling(newAccountName)
+            })
+          ])])
         ])
       ]),
-      addingUser && h(NewUserModal, {
-        adminLabel: 'Owner',
-        userLabel: 'User',
-        title: 'Add user to Billing Project',
-        footer: 'Warning: Adding any user to this project will mean they can incur costs to the billing associated with this project.',
-        addFunction: Ajax().Billing.project(projectName).addUser,
-        onDismiss: () => this.setState({ addingUser: false }),
-        onSuccess: () => this.refresh()
-      }),
-      editingUser && h(EditUserModal, {
-        adminLabel: 'Owner',
-        userLabel: 'User',
-        user: editingUser,
-        saveFunction: Ajax().Billing.project(projectName).changeUserRoles,
-        onDismiss: () => this.setState({ editingUser: false }),
-        onSuccess: () => this.refresh()
-      }),
-      !!deletingUser && h(DeleteUserModal, {
-        userEmail: deletingUser.email,
-        onDismiss: () => this.setState({ deletingUser: false }),
-        onSubmit: _.flow(
-          withErrorReporting('Error removing member from billing project'),
-          Utils.withBusyState(v => this.setState({ updating: v }))
-        )(async () => {
-          this.setState({ deletingUser: false })
-          await Ajax().Billing.project(projectName).removeUser(deletingUser.roles, deletingUser.email)
-          this.refresh()
-        })
-      }),
-      (refreshing || loadingBillingInfo || updatingAccount || updating) && spinnerOverlay
-    ])
-  }
-
-  componentDidUpdate(prevProps, prevState) {
-    const hasBillingScope = Auth.hasBillingScope()
-    if (prevState.hasBillingScope !== hasBillingScope) {
-      this.setState({ hasBillingScope })
-      hasBillingScope && Utils.withBusyState(loading => this.setState({ loading }), this.loadBillingInfo)()
-    }
+      div({
+        style: {
+          marginTop: '1rem',
+          display: 'flex'
+        }
+      }, [
+        h(NewUserCard, {
+          onClick: () => setAddingUser(true)
+        }),
+        div({ style: { flexGrow: 1 } },
+          _.map(member => {
+            return h(MemberCard, {
+              adminLabel: 'Owner',
+              userLabel: 'User',
+              member, adminCanEdit,
+              onEdit: () => setEditingUser(member),
+              onDelete: () => setDeletingUser(member)
+            })
+          }, projectUsers)
```

Comment:
Why don't we need to filter anymore?

Reply:
There's actually no search box rendered on the page, I think the state was a holdover.

Reply:
ah okay